src/app/parcels/fetchParcelTableData.ts

```diff
@@ -131,10 +107,59 @@ type GetParcelDataAndCountResult =
           };
       };
 
+const getParcelsCount = async (
+    supabase: Supabase,
+    filters: Filter<ParcelsTableRow, any>[],
+    abortSignal: AbortSignal
+): Promise<number> => {
+    let query = supabase.from("parcels_plus").select("*", { count: "exact", head: true });
+
+    filters.forEach((filter) => {
+        if (filter.methodConfig.paginationType === PaginationType.Server) {
+            query = filter.methodConfig.method(query, filter.state);
+        }
+    });
+
+    query = query.abortSignal(abortSignal);
+
+    const { count, error } = await query;
+
+    if (error) {
+        const logId = abortSignal.aborted
+            ? await logInfoReturnLogId("Aborted fetch: parcel table count", error)
+            : await logErrorReturnLogId("Error with fetch: parcel table count", error);
+        if (abortSignal.aborted) {
+            throw new AbortError("fetch", "parcel table", "logId");
+        }
+
+        throw new DatabaseError("fetch", "parcel table", logId);
+    }
+
+    if (count === null) {
+        const logId = await logErrorReturnLogId("Error with fetch: Parcels, count is null");
+        throw new DatabaseError("fetch", "parcels", logId);
+    }
+    return count;
+};
+
+const getCongestionChargeDetailsForParcelsTable = async (
```

Comment:
personal preference so up to you but I think I would put this function above the others because it seems less related to the count and data calls

Reply:
Sounds good, done